versions/development/SPEC.md

```diff
@@ -1049,6 +1057,43 @@ output {
 }
 ```
 
+#### Directory Outputs
+
+Directories can be declared as outputs just like files. For example:
+```
+output {
+  Directory d = "created/directory/"
```

Comment:
So if I my workflow/task produces `created/directory` and `created/directory2`, only `created/directory` would appear.  If I wanted both, I would make my output `created/`, correct?

Reply:
if by "appear" you mean "would be saved as an output", then yes, correct!

Reply:
@cjllanwarne Is the trailing `/` purely superfluous here?